consensus/src/dag/reliable_broadcast.rs

```diff
@@ -183,53 +179,51 @@ impl NodeBroadcastHandler {
     }
 }
 
-fn read_signatures_from_storage(
+fn read_votes_from_storage(
     storage: &Arc<dyn DAGStorage>,
     epoch: u64,
-) -> BTreeMap<u64, BTreeMap<Author, NodeDigestSignature>> {
-    let mut signatures_by_round_peer = BTreeMap::new();
+) -> BTreeMap<u64, BTreeMap<Author, Vote>> {
+    let mut votes_by_round_peer = BTreeMap::new();
 
-    let all_node_signatures = storage.get_node_signatures().unwrap_or_default();
+    let all_votes = storage.get_votes().unwrap_or_default();
     let mut to_delete = vec![];
-    for (node_id, node_sig) in all_node_signatures {
+    for (node_id, vote) in all_votes {
         if node_id.epoch() == epoch {
-            signatures_by_round_peer
+            votes_by_round_peer
                 .entry(node_id.round())
                 .or_insert_with(BTreeMap::new)
-                .insert(node_id.author(), node_sig);
+                .insert(node_id.author(), vote);
         } else {
             to_delete.push(node_id);
         }
     }
-    if let Err(err) = storage.delete_node_signatures(to_delete) {
+    if let Err(err) = storage.delete_vote(to_delete) {
```

Comment:
```suggestion
    if let Err(err) = storage.delete_votes(to_delete) {
```
delete_votes plural?